mashina.Inc/build/serverBaseDir_tomcat7/work/Tomcat/localhost/_/org/apache/jsp/HellW_jsp.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Generated by the Jasper component of Apache Tomcat
+ * Version: Apache Tomcat/7.0.78
+ * Generated at: 2019-04-15 17:46:52 UTC
+ * Note: The last modified time of this file was set to
+ *       the last modified time of the source file after
```

Comment:
I believe this file should not be here

Reply:
Seems like they left after multiple changes in file structure, can I just remove them?

Reply:
Yeap, and add build package to .gitignore

Reply:
Just merge the updated branch with gitignore

Reply:
So, we don't need build package in commits?

Reply:
So, let's make a guess why we need not add to git repo all files that were generated by compiling, building and etc.?

Reply:
`Git is a distributed version control system for tracking changes in source code during software development`
Source code != binary machine code.
And if we will store all binary files into git it will produce a lot of trouble and conflicts between different version of the app from all team's member.
Because source code does not depend on your machine configuration, but binary files do